cmd/consts/dymension.go

```diff
@@ -70,7 +71,7 @@ const (
 const (
 	MockHubID       = "mock"
 	LocalHubID      = "dymension_100-1"
-	PlaygroundHubID = "dymension_1405-1"
+	PlaygroundHubID = "dymension_100-1"
```

Comment:
please remove occurances where avail is set as the default da to replace celestia, currently it's not tested/integrated into roller and isn't considered stable